configure.py

```diff
@@ -561,14 +569,24 @@ def add_generated_header(h: str):
 
         for version in versions:
             s_glob = "ver/" + version + "/" + re.sub("src/", "asm/nonmatchings/", c_file)[:-2] + "/*.s"
-            n.build(
-                obj(c_file),
-                "cc_dsl" if status == 0 else "cc",
-                c_file,
-                implicit = None if not args.depend_on_s else glob(s_glob),
-                order_only="generated_headers_" + version,
-                variables={ "version": version }
-            )
+            if c_file.startswith("src/ultra/"):
```

Comment:
It'd be nice to isolate the change to this if/else and do the n.build afterward, using a variable that gets set.

So like if it's a file in src/ultra, set the rule to "cc_kmc", else if status == 0, set it to "cc_dsl", else "cc"

Reply:
Fixed